src/main/java/Kitty.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+
+public class Kitty {
+    public static final int MAX_TASKS = 100;
+    public static Task[] tasks = new Task[MAX_TASKS];
+    public static int totalTasksCount = 0;
+
+    //Methods
+    public static void printList(Task[] task) {
+        System.out.println();
+        System.out.println("Here are the tasks you have!");
+        for (int i = 0; i < totalTasksCount; i++) {
+            System.out.print(i+1 + ".");
+            System.out.println(task[i]);
+        }
+        System.out.println();
+        System.out.println(" |\\__/,|   (`\\\n" +
+                            " |_ _  |.--.) )\n" +
+                            " ( T   )     /\n" +
+                            "(((^_(((/(((_/");
+        System.out.println("===============================================");
```

Comment:
Perhaps you could make the ASCII art a constant instead of in-lining it in the code.